### PR TITLE
Add python-path parameter 

### PR DESCRIPTION
**Pull request by titulebolide**
Add python-path parameter to specify (optionally) the path of the python3 executable (to chose for example for a virtual env)

### CODE REVIEW

**Comment by titulebolide** [comment]
What does this mean?

**Comment by titulebolide** [comment]
Ho - okay should i be running black formatter before pushing?

**Comment by lorcalhost** [comment]
The checks fail because of formatting, check [here](https://github.com/lorcalhost/BTB-manager-telegram/blob/main/CONTRIBUTING.md).

Also, `{settings.PYTHON_PATH}` should also be used in other parts of the program such as the update functions.

**Comment by titulebolide** [comment]
Oh yeah i've forgot about the other runs...

**Comment by titulebolide** [comment]
And also, shutil.which('python3') will be using the manager's python, so if it is running in a venv this does not happen correctly...
shutil.which('python3') should be used here:
https://github.com/lorcalhost/BTB-manager-telegram/blob/0f6049fb2b77f97c039b53c11108835ccfe19eae/btb_manager_telegram/handlers.py#L404
instead of `$(which python3)` that would make BTB manager restart out of its eventual venv

**Comment by titulebolide** [comment]
Actaully it could be the awaited behaviour that the manager launches by default BTB with the same python exectuable as it is running with?

**Comment by lorcalhost** [comment]
Expected behavior is: the script kills it's own process and then restarts it after a git pull  
So it should be re-launched with venv's python

**Comment by titulebolide** [comment]
Yes but in the cas the TG bot has its own venv, i think that the expected behaviour for every interaction with BTB bot if the python path is unset is the BTB to be ran with TG's venv

**Comment by lorcalhost** [comment]
Ah you mean with which executable we are supposed to launch binance-trade-bot.  
I guess people would be using a single venv for both programs right? In this case just launch with venv.  
What would you do if you had to install both bots using venv? Use one or two?

**Comment by titulebolide** [comment]
I would be using separate venv

**Comment by titulebolide** [comment]
Then for everything TG-related it uses sys.executable, and for everything BTB related it uses settings.PYTHON_PATH that defaults to sys.executable if unset

**Comment by lorcalhost** [comment]
Alright this seems to be working as expected on my end, are you having an issues on this PR?

**Comment by titulebolide** [comment]
on my end it is good

File: README.md [hunk 1]
```diff
@@ -95,6 +98,8 @@ If _Binance Trade Bot_ and _BTB-Manager-Telegram_ were **not** installed in the
 ```console
 optional arguments:
   -p PATH, --path PATH  (optional) binance-trade-bot installation absolute path
+  -y PYTHON, --python PYTHON_PATH
+                        (optional) python binary to be used for the BTB. If unset, uses the same executable (and thus virtual env if any) than the telegram bot.
```
[the remark this answers is not in the document]
Comment (titulebolide) [reply README.md:102]: why not, but i think ppl understant what BTB refers to since it is in the name of this repo

**Comment by lorcalhost** [comment]
Everything seems ok, tomorrow morning I will have a last check and merge 😄

**Comment by lorcalhost** [comment]
@all-contributors please add @titulebolide for code, idea

**Pull request merged**